src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.css

```diff
@@ -0,0 +1,35 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  margin-top: 15px;
+  margin-bottom: 100px;
+}
+
+.title {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+}
+
+.description {
+  margin-bottom: 18px;
+  font-size: var(--size-tiny);
+  line-height: 18px;
+  color: color-mod(var(--temp-grey-blue-7) alpha(85%));
+}
+
+.buttonGroup {
+  margin-top: 32px;
+}
+
+.amount {
+  align-self: flex-end;
+  margin-bottom: 2px;
+  font-size: var(--size-tiny);
+  line-height: 18px;
+  color: color-mod(var(--temp-grey-blue-7) alpha(85%));
+}
```

Comment:
Amount should be bold

Spec
![Screenshot from 2021-04-20 13-55-52](https://user-images.githubusercontent.com/1193222/115387737-bae2b580-a1e3-11eb-9965-8514f19abb74.png)

Yours
![Screenshot from 2021-04-20 13-55-50](https://user-images.githubusercontent.com/1193222/115387750-bfa76980-a1e3-11eb-9c1b-a2b1f87a1ba3.png)